### PR TITLE
Improved Gambeson

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Adds a kevlar version of the gambeson and XL gambeson."

#### Purpose of change
These are intended as modernized versions of the gambeson, similar to kevlar motorcycle kit. It adds significant ballistic protection while keeping the existing protection of the standard gambeson at a higher skill and material cost.

#### Describe the solution
This uses the amount of layered kevlar from other full body armors like the survivor suits.

#### Describe alternatives you've considered
Dying in a hail of gunfire.

#### Testing
Spawned both items, wore them. Spawned book and necessary crafting materials, crafted each one.

#### Additional context
This could be extended into removing the gambeson from the chainmail hauberk and suit, allowing for more mix and match combinations.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
I think you've got two copies of the recipes, not the recipes and the actual items.

**Comment by Rivet-the-Zombie** [comment]
I think you've got two copies of the recipes, not the recipes and the actual items.

**Comment by Drew4484** [comment]
Updated to fix the double recipe.

File: data/json/items/armor/suits_protection.json [hunk 1]
```diff
@@ -594,6 +594,25 @@
     "flags": [ "VARSIZE", "STURDY" ],
     "armor": [ { "encumbrance": 15, "coverage": 80, "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ] } ]
   },
+   {
+    "id": "k_gambeson",
+    "type": "ARMOR",
+    "name": { "str": "gambeson" },
```
Comment (casswedson) [inline data/json/items/armor/suits_protection.json:600]:
```suggestion
    "name": { "str": "Kevlar gambeson" },
```

File: data/json/items/armor/suits_protection.json [hunk 2]
```diff
@@ -602,6 +621,14 @@
     "proportional": { "weight": 1.125, "volume": 1.13, "price": 1.25 },
     "flags": [ "VARSIZE", "STURDY", "OVERSIZE" ]
   },
+  {
+    "id": "xl_k_gambeson",
+    "type": "ARMOR",
+    "name": { "str": "XL kevlar gambeson" },
```
Comment (casswedson) [inline data/json/items/armor/suits_protection.json:628]:
```suggestion
    "name": { "str": "XL Kevlar gambeson" },
```

File: data/json/items/armor/suits_protection.json [hunk 3]
```diff
@@ -594,6 +594,25 @@
     "flags": [ "VARSIZE", "STURDY" ],
     "armor": [ { "encumbrance": 15, "coverage": 80, "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ] } ]
   },
+   {
+    "id": "k_gambeson",
+    "type": "ARMOR",
+    "name": { "str": "gambeson" },
+    "description": "A thick jacket of quilted fabric, designed to be worn underneath mail or other armor. This one incorporates kevlar layers added for better protection.",
```
Comment (casswedson) [inline data/json/items/armor/suits_protection.json:601]:
```suggestion
    "description": "A thick jacket of quilted fabric, designed to be worn underneath mail or other armor.  This one incorporates Kevlar layers added for better protection.",
```

File: data/json/items/armor/suits_protection.json [hunk 4]
```diff
@@ -594,6 +594,26 @@
...
+	'weight": "453 g"
```
Comment (casswedson) [inline data/json/items/armor/suits_protection.json:602]: It should have a weight based on the amount of kevlar used, not the weight of a regular gambeson

Comment (Drew4484) [reply data/json/items/armor/suits_protection.json:602]: Raised the cost up to 36 layered kevlar panels, that gives a weight of 2.880 kg and 2.52 L, which is close to the gambeson's 3 L. 

 It looks like the gambeson itself is off however, it's 453 g, which doesn't add up based on the material, 26 rags at 80 g should be 2080 g.

File: data/json/items/armor/suits_protection.json [hunk 5]
```diff
@@ -594,6 +594,26 @@
     "flags": [ "VARSIZE", "STURDY" ],
     "armor": [ { "encumbrance": 15, "coverage": 80, "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ] } ]
   },
+   {
+    "id": "k_gambeson",
+    "type": "ARMOR",
+    "name": { "str": "kevlar gambeson" },
+    "description": "A thick jacket of quilted fabric, designed to be worn underneath mail or other armor.  This one incorporates Kevlar layers added for better protection.",
+	'weight": "453 g"
+    "volume": "3 L",
```
Comment (casswedson) [inline data/json/items/armor/suits_protection.json:603]:
```suggestion
	  "weight": "453 g",
    "volume": "3 L",
```

**Pull request merged**